requirements.txt

```diff
@@ -1 +1 @@
-ansible==2.1.0
+ansible==2.2.0
```

Comment:
@miabbott Good point about the bug fixes.  Does it make sense to use ansible>=2.2.0 to be more explicit about the requirements?